editor/filesystem_dock.cpp

```diff
@@ -751,6 +751,14 @@ void FileSystemDock::_navigate_to_path(const String &p_path, bool p_select_in_fa
 			}
 		}
 	}
+
+	// Ensure that the FileSystem dock is visible.
+	if (get_window() == get_tree()->get_root()) {
+		TabContainer *tab_container = (TabContainer *)get_parent_control();
+		tab_container->set_current_tab(tab_container->get_tab_idx_from_control((Control *)this));
+	} else {
+		get_window()->grab_focus();
+	}
```

Comment:
I wonder if it should be here or in the public `navigate_to_path()`.
Although from my testing it doesn't affect the current behavior, so it's probably fine.

Reply:
The public one ends up calling the private, so I figured it was best to keep it at the lowest level of that operation chain.

Reply:
The call in EditorAudioBuses should be changed to `navigate_to_path()`.

The EditorInterface call should be left as is, to keep the current behavior. We could expose `navigate_to_path()` as a new method, but that's out of scope for this PR.

The call in EditorNode's `load_docks()` should not focus file dock.

That's all outside calls I found, and it convinces me more that only the public method should be changed.

Reply:
Alright, I'll move that focus bit to the public, thanks